.github/workflows/linux-workflow.yml

```diff
@@ -53,6 +53,16 @@ jobs:
             cmakeflags: -DCMAKE_DISABLE_PRECOMPILE_HEADERS=ON
             detail: -nopch
             experimental: false
+          - os: ubuntu-18.04
+            platform: x86
+            compiler: gcc
+            detail: -appimage
+            experimental: false
+          - os: ubuntu-18.04
+            platform: x64
+            compiler: gcc
+            experimental: false
+            detail: -appimage 
```

Comment:
I'm wondering if we want to set `experimental` to `true` on these jobs so that if they fail, it won't fail the entire workflow via `continue-on-error`

Reply:
The appimage is set to continue on fail, so if it does not build the other elements of the matrix can succeed.